features/app_and_device_attributes.feature

```diff
@@ -0,0 +1,45 @@
+Feature: App and Device attributes present
+
+Scenario: App and Device info is as expected
+    When I run "AppAndDeviceAttributesScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the "Bugsnag-API-Key" header equals "12312312312312312312312312312312"
+    
+    # Device
+    
+    And the payload field "events.0.device.osName" equals "iOS"
+    And the payload field "events.0.device.jailbroken" is false
+    And the payload field "events.0.device.osVersion" matches the regex "\d+\.\d+"
+    And the payload field "events.0.device.manufacturer" equals "Apple"
+    And the payload field "events.0.device.locale" is not null
+    And the payload field "events.0.device.id" is not null
+    And the payload field "events.0.device.model" equals "iPhone10,4"
+    And the payload field "events.0.device.modelNumber" is not null
+    And the payload field "events.0.device.runtimeVersions.osBuild" is not null
+    And the payload field "events.0.device.runtimeVersions.clangVersion" is not null 
+    And the payload field "events.0.device.totalMemory" is an integer
+
+    # DeviceWithState
+
+    And the payload field "events.0.device.freeDisk" is an integer
+    And the payload field "events.0.device.freeMemory" is an integer
+    #And the payload field "events.0.device.orientation" equals "portrait"
```

Comment:
Don't leave lines commented out.  If a check fails and it can't be easily resolved, remove it and raise an internal ticket to address the failure and reinstate the check.

Reply:
Ticket raised, line left commented in for context, on the assumption that this needs addressed before UAT is considered complete.